cmd/kubeadm/app/cmd/phases/workflow/runner.go

```diff
@@ -309,6 +309,7 @@ func (e *Runner) BindToCommand(cmd *cobra.Command) {
 	phaseCommand := &cobra.Command{
 		Use:   "phase",
 		Short: fmt.Sprintf("use this command to invoke single phase of the %s workflow", cmd.Name()),
+		// TODO: this logic is currently lacking verification if a subphase name is valid!
```

Comment:
Please remove this.
As agreed we will tackle args management when all the phases are merged and I don't want this discussion leaking in all the PRs

Reply:
+1, plus it doesn't really apply, validation is actually performed if you mistype a phase name.